### DISCUSSION TITLE
Geo Blocked

### DISCUSSION THREAD

**Discussion by lexieye** [iptv-org/community#406] [Q&A] [answered]
I live in the uk and have downloaded the uk m3u playlist with 200 channels from this repo
I am using an original Firestick with the PVR IPTV Simple Client 

I don’t understand how I’m geoblocked? Is it my internet provider

**Comment by betweenrows** [comment, marked as answer]
Which channels?  Several in the playlist no longer work, unfortunately.  ITV1, ITV2, More4, Really, etc.
